be/src/column/json_column.cpp

```diff
@@ -0,0 +1,57 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "column/json_column.h"
+
+#include "util/hash_util.hpp"
+#include "util/mysql_row_buffer.h"
+
+namespace starrocks::vectorized {
+
+int JsonColumn::compare_at(size_t left, size_t right, const starrocks::vectorized::Column& rhs,
+                           int nan_direction_hint) const {
+    JsonValue* x = get_object(left);
+    const JsonValue* y = rhs.get(right).get_json();
+    return x->compare(*y);
+}
+
+void JsonColumn::fnv_hash(uint32_t* hash, uint32_t from, uint32_t to) const {
+    for (uint32_t i = from; i < to; i++) {
+        JsonValue* json = get_object(i);
+        int64_t h = json->hash();
+        hash[i] = HashUtil::fnv_hash(&h, sizeof(h), hash[i]);
```

Comment:
fnv_hash is used for bucket shuffling, json->hash() already compute out a hash value, so use the high or low   32 bit is just ok. HashUtil::fnv_hash is needless.

Reply:
The `fnv_hash` needs to take the input hash into account, so I could not use `json->hash` as the result?